modules/ROOT/pages/tuning-guide.adoc

```diff
@@ -0,0 +1,25 @@
+= Performance Tuning Guide
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Running Mule applications in their optimally performing state requires a careful review of not only the applications but also the environment where they run. Mule 4 was designed to tune itself for optimal performance depending on several factors; however, this does not prevent your applications from experiencing performance issues because of their construction or dependencies. Similarly, for on-premises installations, the environment itself can be tuned so the Mule application can take full advantage of it. Many variables base the performance of your application, and tuning these variables still requires a trial-and-error process. +
+To make this process easier, the tuning guide highlights performance best practices and the ideal test environments to run applications. These recommendations are collected from the Services Engineering team,  in-house benchmarking efforts by the MuleSoft Performance Engineering, and from the Development team.
+
+== Performance Tuning Recommendations
+
+Optimize the performance of your Mule apps by applying tuning recommendations at the application level.
+
+See xref:tuning-recommendations.adoc[Performance Tuning Recommendations] for details.
+
+== Performance Test Validations
+
+Learn about the prerequisites and best practices before you start testing the performance of your Mule apps.
+
+See xref:tuning-test-validations.adoc[Performance Test Validations] for details.
+
+== Performance Monitoring
+
+Monitor the resource consumed by the Mule runtime engine (Mule) and know what resources have been exhausted during the tests.
+
+See xref:tuning-monitoring.adoc[Performance Monitoring] for details.
```

Comment:
See suggestion above for consolidating these into one subtopic, depending on whether this whole text is followed by peer subtopics.
If not, consider directly following "The following recommendations come from the Development and Services Engineering teams and benchmarking efforts by MuleSoft Performance Engineering:" with the list?